addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -236,16 +266,20 @@ class AwsCfnService extends Service {
 
     const awsAccountsService = await this.service('awsAccountsService');
     const accountsList = await awsAccountsService.list();
-
+    // const stackOutputs = ['externalId', 'vpcId', 'subnetId', 'encryptionKeyArn', 'cfnStackId', 'roleArn'];
     const newStatus = {};
     const errors = {};
     const idList = accountsList.forEach(account => account.accountId);
     let res;
     let errorMsg = '';
 
     const checkPermissions = async account => {
-      if (account.cfnStackName === '') {
-        res = 'NEEDSONBOARD';
+      errorMsg = '';
+      if (
+        account.permissionStatus === 'NEEDSONBOARD' ||
+        account.permissionStatus === 'PENDING' // Backend workflow will bring the account out of PENDING
+      ) {
+        res = account.permissionStatus;
         errorMsg = `Account ${account.accountId} needs to be onboarded.`;
```

Comment:
it could go either way--my logic was that the method is trying to check permissions, so accounts that are being handled by the workflow can't be checked, and thus it's like an error. I could probably remove this line and it would be better, since it's sort of a weird response type.

Reply:
removed it 👍🏻